gammapy/data/observation_summary.py

```diff
@@ -125,3 +128,210 @@ def show_in_browser(self):
+        """Plot background as a function of livetime
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes` or None, optional.
+            The `~matplotlib.axes.Axes` object to be drawn on.
+
+        Returns
+        -------
+        ax : `~matplolib.axes`
+            Axis
+        """
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+        ax.plot(self.livetime.to(u.h), self.background, "o", **kwargs)
+
+        ax.set_xlabel('Livetime ({0})'.format(u.h))
+        ax.set_ylabel('Background')
+        ax.axis([0., np.amax(self.livetime.to(u.h).value) * 1.2,
+                 0., np.amax(self.background) * 1.2])
+        ax.set_title('Background evolution')
+        return ax
+
+    def plot_gamma_rate(self, ax=None, **kwargs):
+        """Plot gamma rate for each observation
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes` or None, optional.
+            The `~matplotlib.axes.Axes` object to be drawn on.
+
+        Returns
+        -------
+        ax : `~matplolib.axes`
+            Axis
+        """
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+        labels = list()
+        values = list()
+        for index in range(len(self.gamma_rate)):
+            labels.append(str(int(self.obs_id[index])))
+            values.append(index + 0.5)
+
+        ax.plot(values, self.gamma_rate, "o", **kwargs)
+        ax.set_xlabel('Observation Ids')
+
+        ax.set_xticks(values)
+        ax.set_xticklabels(labels, rotation=-22.5)
+        ax.set_ylabel('$\gamma$ rate ({})'.format(self.gamma_rate.unit))
+        ax.axis([0, len(self.gamma_rate),
+                 0., np.amax(self.gamma_rate.value) * 1.2])
+        ax.set_title('$\gamma$ rates')
+        return ax
+
+    def plot_background_rate(self, ax=None, **kwargs):
+        """Plot background rate for each observation
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes` or None, optional.
+            The `~matplotlib.axes.Axes` object to be drawn on.
+
+        Returns
+        -------
+        ax : `~matplolib.axes`
+            Axis
+        """
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+        labels = list()
+        values = list()
+        for index in range(len(self.bg_rate)):
+            labels.append(str(int(self.obs_id[index])))
+            values.append(index + 0.5)
+
+        ax.plot(values, self.bg_rate, "o", **kwargs)
+        ax.set_xlabel('Observation Ids')
+
+        ax.set_xticks(values)
+        ax.set_xticklabels(labels, rotation=-22.5)
+        ax.set_ylabel('Background rate ({})'.format(self.bg_rate.unit))
+        ax.axis([0, len(self.bg_rate),
+                 0., np.amax(self.bg_rate.value) * 1.2])
+        ax.set_title('Background rates')
+        return ax
+
+    def __str__(self):
+        self.summary()
```

Comment:
There's one test that's still failing because you don't return the string here:
https://travis-ci.org/gammapy/gammapy/jobs/137196513#L944

Either put this:
```
def __str__(self):
    return self.summary()
```
or, if the summary method doesn't have any arguments to configure the summary, you can also get rid of it and just put it's implementation directly in `__str__`.